docs/src/main/docs/security/05_extensibility.adoc

```diff
@@ -0,0 +1,117 @@
+///////////////////////////////////////////////////////////////////////////////
+
+    Copyright (c) 2019 Oracle and/or its affiliates. All rights reserved.
+
+    Licensed under the Apache License, Version 2.0 (the "License");
+    you may not use this file except in compliance with the License.
+    You may obtain a copy of the License at
+
+        http://www.apache.org/licenses/LICENSE-2.0
+
+    Unless required by applicable law or agreed to in writing, software
+    distributed under the License is distributed on an "AS IS" BASIS,
+    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+    See the License for the specific language governing permissions and
+    limitations under the License.
+
+///////////////////////////////////////////////////////////////////////////////
+
+= Extending Security
+
```

Comment:
I started reviewing this, but then realized that this content might be the old content under "Developer’s Guide" at https://helidon.io/docs/latest/#/security/04_tools.  If that's indeed the case, and if there are no changes to be reviewed, then I won't review any further. Pls confirm, or let me know the changes that I must review.

Reply:
There were some changes in Framework integration

Reply:
> 
> 
> There were some changes in Framework integration

I reviewed that part. It looks ok, except for a minor issue for which I've posted a comment.